### PR TITLE
OADP-2380: updating velero table and some tidying up

### PR DESCRIPTION
**Pull request by anarnold97**
### PR

* [OADP-2380](https://issues.redhat.com/browse/OADP-2380)

### Link to docs preview:

* [Moving Install OADP Operator from a module to a page](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator)
    * [Adding Velero table](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator#velero-oadp-version-relationship_installing-oadp-operator)
* [Removing Installing OADP Operator module from renamed Configuring the OpenShift API for Data Protection with Amazon Web Services page](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws)
* [Removing Installing OADP Operator module from renamed Configuring the OpenShift API for Data Protection with Microsoft Azure page](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure)    
* [Removing Installing OADP Operator module from renamed Configuring the OpenShift API for Data Protection with Google Cloud Platform page](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp)
* [Removing Installing OADP Operator module from renamed configuring the OpenShift API for Data Protection with Multicloud Object Gateway page](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg)
* [Removing Installing OADP Operator module from renamed configuring the renamed configuring the OpenShift API for Data Protection with OpenShift Data Foundation  page](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs)

* Renaming the TOC to remove **Installing** from menu:
image

* [Updating Troubleshooting to include Velero relationship table](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/troubleshooting#velero-oadp-version-relationship_oadp-troubleshooting)

* [Moving Uninstalling to new section in the TOC](https://65904--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/uninstalling-oadp)

image

  

### Version(s):

* OADP 1.1 & 1.2

##### Openshift
Enterprise 4.11 → branch/enterprise-4.11
Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14

#### Issue:

* https://issues.redhat.com/browse/OADP-2380

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@weshayutin - please can you review and give me a LGTM if happy

I prefer the full table including the Z-streams for clarity. Moving it to a single table will make things easier

There are a few tidy-up items included. Nothing major

**Review by weshayutin — APPROVED** [review]
Thanks @anarnold97

File: backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+:_content-type: PROCEDURE
+[id="oadp-installing-operator-doc"]
+= Installing the OADP Operator
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-operator
+
+toc::[]
+
+You install the OpenShift API for Data Protection (OADP) Operator on {product-title} {product-version} by using Operator Lifecycle Manager (OLM).
```
Comment (skrthomas) [inline backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc:9]:
```suggestion
You can install the OpenShift API for Data Protection (OADP) Operator on {product-title} {product-version} by using Operator Lifecycle Manager (OLM).
```

**Review by skrthomas — COMMENTED** [review]
One nit, otherwise LGTM

**Comment by anarnold97** [comment]
@skrthomas - thanks very much. change made (cos it looked weird, right?)

**Review by sheriff-rh — APPROVED** [review]
LGTM! Merging.

**Pull request merged**